src/Global.h.Rt

```diff
@@ -169,6 +169,61 @@ int kbhit(void);
 #define DEBUG_H 1
 #endif
 
+/// Allocation of memory for an FTabs
+void FTabs::Alloc(int nx,int ny,int nz) {
```

Comment:
I think this is not used

Reply:
Closed in 2c27d9263125e8726e2bb5c921b47491b6e4690c